tokio-signal/examples/ctrl-c.rs

```diff
@@ -1,42 +0,0 @@
-#![warn(rust_2018_idioms)]
-#![feature(async_await)]
-
-use futures_util::stream::StreamExt;
-
-/// how many signals to handle before exiting
-const STOP_AFTER: u64 = 10;
-
-#[tokio::main]
-async fn main() {
```

Comment:
Maybe sticking with the crate docs is enough, but I'm just curious, is there a plan to keep around any form of signal specific examples?

Reply:
could bring them back in the main tokio crate once it is re-exported. There is nothing blocking that really, so if you want to add a `tokio::signal` mod that exports a minimal set of functionality, we can go w/ that. Maybe start w/ `tokio::signal::ctrl_c()`?

Reply:
mostly, I thought it was odd to only have signal examples in tokio-net, so I took the short path of removing them for now...

Reply:
Sounds good